src/main.rs

```diff
@@ -277,6 +284,20 @@ enum SemverChecksCommands {
     CheckRelease(CheckRelease),
 }
 
+#[derive(Copy, Clone, ValueEnum, Debug)]
+enum LintLevel {
+    Major,
+    Minor,
+}
+impl std::fmt::Display for LintLevel {
```

Comment:
FYI you can use `ValueEnum` to implement this: https://github.com/clap-rs/clap/blob/master/src/util/color.rs#L67